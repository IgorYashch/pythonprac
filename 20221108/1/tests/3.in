a = DivStr("jaldsflkdsafjlkdsajflkdsajlkfdsajlkfjlkdsajf;lkdsajf;ldsajflkjdsalkjkdsfjkksdjdsjjheroihewoiufbkjdnvijerwnckdmscjbgijenfkjvbdkjvfbkdfjblkvjsbfjvfdbkjhvbsgnbsdhfjbhfdsjvbjfdsbvhjfdsbhvfdj bvhjsb")

print(* a // 10, a % 10, sep=', ', end='|\n')