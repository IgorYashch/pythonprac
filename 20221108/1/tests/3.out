jaldsflkdsafjlkdsaj, flkdsajlkfdsajlkfjl, kdsajf;lkdsajf;ldsa, jflkjdsalkjkdsfjkks, djdsjjheroihewoiufb, kjdnvijerwnckdmscjb, gijenfkjvbdkjvfbkdf, jblkvjsbfjvfdbkjhvb, sgnbsdhfjbhfdsjvbjf, dsbvhjfdsbhvfdj bvh, jsb|
